apps/admin-portal/src/store/reducers/identityProvider.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {IdentityProviderReducerStateInterface} from "../../models";
+import {IdentityProviderActions, IdentityProviderActionTypes} from "../actions/types/identityProvider";
```

Comment:
Shall we make the file lowercase ?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/632/commits/e0c193e52bcf9374036f406699147fbc98e8bd77.